recipe/meta.yaml

```diff
@@ -15,11 +15,12 @@ source:
 build:
   noarch: python
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
   build:
     - python
+    - pip
```

Comment:
I think python and pip should be in `host`, rather than build, like in the example recipe in staged recipes: https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L39

Reply:
I was following the link given in https://github.com/conda-forge/python-sounddevice-feedstock/pull/1#issuecomment-387140551, which uses `build` in the example, but as I just saw it mentions "`build` or `host`" in the text below.
If you prefer I can move them to `host`.

Reply:
I think it would be nice to move it to host, while we are thinking about it. Also, please feel free to add yourself to the maintainers list, if you are interested